google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/LoadConfiguration.java

```diff
@@ -70,6 +70,17 @@
 
 
     /**
+     * Sets the string that represents a null value in a CSV file. For example, if you specify "\N",
+     * BigQuery interprets "\N" as a null value when loading a CSV file. The default value is the
+     * empty string. If you set this property to a custom value, BigQuery throws an error if an
+     * empty string is present for all data types except for {@code STRING} and {@code BYTE}. For
+     * {@code STRING} and {@code BYTE} columns, BigQuery interprets the empty string as an empty
+     * value.
+     */
+    Builder setNullMarker(String nullMarker);
```

Comment:
Why not add the field to the `CsvOptions` class? In the issue description, null marker is only available when CSV format is used and it should be a part of format options.